### PR TITLE
Update network configuration section in "HowToUseIOTjs".

### PR DESCRIPTION
**Pull request by ziransun**
Signed-off-by: Ziran Sun <ziran.sun@samsung.com>

### CODE REVIEW

File: docs/HowToUseIoTjs.md [hunk 1]
```diff
@@ -51,8 +51,23 @@ Then place your script(s) in tools/fs/contents, for instance:
...
+You need to run "make menuconfig" again and edit credentials.
+make menuconfig  
+    # Application Configuration  --->  
+    #  Examples  --->  
+    #   [*] IoT.js StartUp example  
+    #     (/rom/example/index.js) Main javascript file  
+    #     [*]   Connect WiFi
+    #     ("APSSID") SSID of AP
+    #     ("APPassword") Passphrase of AP    
+    #     (0)   Authentication type
```
Comment (rzr) [inline docs/HowToUseIoTjs.md]: In this case it would make sense to change values above and below:

File: docs/HowToUseIoTjs.md [hunk 2]
```diff
@@ -51,8 +51,23 @@ Then place your script(s) in tools/fs/contents, for instance:
 WiFi can be also enabled on boot if you want to join a network 
 before running the script.
 
-You need to run menuconfig again and edit credentials,
-(typically for WPA2: Auth=5 and Enc=4)
+You need to run "make menuconfig" again and edit credentials.
+make menuconfig
```
Comment (rzr) [inline docs/HowToUseIoTjs.md]: indent with 4 spaces and skip line
just like above

**Comment by seinfra** [comment]
Target : [0bdb61eba23bba2fbace99f4d9b1e33c86c4de9f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0bdb61eba23bba2fbace99f4d9b1e33c86c4de9f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [770b032c9ee95783da34f49453dcd2be835eb181] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@ziransun This PR has 3 commits, two has a perfectly same title, another is modifying for modifying.
It is not allowed. These 3 commits has a same purpose. Let's squash or fixup them thru rebase command.

**Comment by seinfra** [comment]
Target : [770b032c9ee95783da34f49453dcd2be835eb181] - Code Rule Check OK.

File: docs/HowToUseIoTjs.md [hunk 3]
```diff
@@ -51,8 +51,23 @@ Then place your script(s) in tools/fs/contents, for instance:
 WiFi can be also enabled on boot if you want to join a network 
 before running the script.
 
-You need to run menuconfig again and edit credentials,
-(typically for WPA2: Auth=5 and Enc=4)
+You need to run "make menuconfig" again and edit credentials.
```
Comment (rzr) [inline docs/HowToUseIoTjs.md]: just insert CR here

**Comment by seinfra** [comment]
Target : [2e5199d959961c5d33115657b83f6331da879b7d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2e5199d959961c5d33115657b83f6331da879b7d] - Code Rule Check (C++) OK.

**Comment by rzr** [comment]
You need to:

1/ rebase on upstream master

2/ squash commits

3/ push to your branch:

4/ verify formating on

https://github.com/ziransun/TizenRT/blob/master/docs/HowToUseIoTjs.md

5/ and ask for review here

Good luck

**Comment by sunghan-chang** [comment]
@ziransun Here is no activity. Let's close this.

**Pull request closed without merging**